data/abilities/execution/60f63260-39bb-4136-87a0-b6c2dca799fc.yml

```diff
@@ -14,7 +14,8 @@
           $wc=New-Object System.Net.WebClient;
           $output="PowerShellCore.msi";
           $wc.DownloadFile("https://github.com/PowerShell/PowerShell/releases/download/v6.2.2/PowerShell-6.2.2-win-x64.msi", $output);
-          msiexec.exe /package PowerShellCore.msi /quiet ADD_EXPLORER_CONTEXT_MENU_OPENPOWERSHELL=1 ENABLE_PSREMOTING=1 REGISTER_MANIFEST=1;
```

Comment:
@khyberspache can you confirm this?

Reply:
Yes, confirmed. If the PowerShell command does not wait for the installation to finish or if the installation takes a bit longer than expected, the pwsh.exe executable is not found, despite being in the path, because it hasn't been installed and put in the directory.